iOS/Flows/Conversation/Threads/Conversation Collection Views/Cells/Message/MessageTextView.swift

```diff
@@ -25,13 +25,13 @@ class MessageTextView: TextView {
     }
 
     func set(text: Localized, messageContext: MessageContext) {
-        let textColor: Color = messageContext == .status ? .darkGray : .white
+        let textColor: Color = messageContext == .status ? .darkGray : .textColor
```

Comment:
Changed text color to black and used it here.

Reply:
nice